src/command_modules/azure-cli-network/azure/cli/command_modules/network/_template_builder.py

```diff
@@ -219,34 +219,37 @@ def build_load_balancer_resource(name, location, tags, backend_pool_name, fronte
         ],
         'frontendIPConfigurations': [frontend_ip_config]
     }
-
     lb = {
         'type': 'Microsoft.Network/loadBalancers',
         'name': name,
         'location': location,
         'tags': tags,
-        'apiVersion': '2015-06-15',
+        'apiVersion': get_api_version(ResourceType.MGMT_NETWORK),
         'dependsOn': [],
         'properties': lb_properties
     }
+    if sku and supported_api_version(ResourceType.MGMT_NETWORK, min_api='2017-08-01'):
```

Comment:
I don't think we need the api range check here, because that is already done in the `_params.py`.
If agreed, please apply the same simplification on 2 other places in the PR.